Calculator/Calculator/Formula.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Formula.swift
+//  Calculator
+//
+//  Created by 천승현 on 2023/01/27.
+//
+struct Formula {
+    private (set) var operands = CalculatorItemQueue<Double>()
+    private (set) var operators = CalculatorItemQueue<Operator>()
```

Comment:
접근제어 좋습니다 👏

일반적으로 괄호를 붙여서 사용합니다!
```suggestion
    private(set) var operands = CalculatorItemQueue<Double>()
    private(set) var operators = CalculatorItemQueue<Operator>()
```